### PR TITLE
filter north star charts by experiments in current project

### LINKED ISSUE #785: [Bug] North Star gantt chart should only include experiments in the current project
### Summary

The North Star graph on the management dashboard always includes all experiments, even if a specific project is selected.

It should filter the list to the currently selected project (if there is one), just like the other dashboard components.

### PR DESCRIPTION
**Pull request by bryce-fitzsimons**
Closes #785

### CODE REVIEW

File: packages/front-end/components/HomePage/Dashboard.tsx [hunk 1]
```diff
@@ -11,23 +11,27 @@ import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
...
+    : experiments;
 
   return (
     <>
       <div className={"container-fluid dashboard p-3 " + styles.container}>
         <h1>Hello {name}</h1>
         <div className="row">
           <div className="col-md-12">
-            <NorthStar experiments={experiments} />
+            <NorthStar experiments={northStarExperiments} />
```
Comment (jdorn) [inline packages/front-end/components/HomePage/Dashboard.tsx:34]: This isn't the experiment list used in the gannt chart.  The `<NorthStarMetricDisplay>` component does it's own API call to get the list of experiments.

**Review by jdorn — CHANGES_REQUESTED** [review]

File: packages/front-end/components/HomePage/Dashboard.tsx (hunk 1 above)
Comment (bryce-fitzsimons) [reply packages/front-end/components/HomePage/Dashboard.tsx]: whoops, fixed (I hope)

File: packages/front-end/components/HomePage/NorthStar.tsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
 
 const NorthStar: FC<{
   experiments: ExperimentInterfaceStringDates[];
-}> = ({ experiments }) => {
+  project: string;
+}> = ({ experiments, project }) => {
```
Comment (jdorn) [inline packages/front-end/components/HomePage/NorthStar.tsx:16]: You can avoid all these new props.  The currently selected project is available via context anywhere in the app.

```ts
const {project} = useDefinitions()
```

**Review by bttf — APPROVED** [review]
👍🏾

**Review by jdorn — APPROVED** [review]

**Pull request merged**